ReClass.NET/Forms/MainForm.cs

```diff
@@ -63,21 +63,25 @@ public MainForm()
 
 			InitializeComponent();
 
-			Text = $"{Constants.ApplicationName} ({Constants.Platform})";
+			string randomWindowTitle = Utils.RandomString(Program.GlobalRandom.Next(15, 20));
```

Comment:
Could you replace the three changes with a single (inline) method? Possible implementation:
```csharp
void SetWindowTitle(string extra = null)
{
	var title = $"{(Program.Settings.RandomizeWindowTitle ? Utils.RandomString(Program.GlobalRandom.Next(15, 20)) : Constants.ApplicationName)} ({Constants.Platform})";
	if (!string.IsNullOrEmpty(extra))
	{
		title += $" - {extra}";
	}
	Text = title;
}
```